src/lbfgs.jl

```diff
@@ -89,10 +89,16 @@ function LBFGSSolver(nlp::M; mem::Int = 5) where {T, V, M <: AbstractNLPModel{T,
   return LBFGSSolver{T, V, Op, M}(x, xt, gx, gt, d, H, h)
 end
 
-function LinearOperators.reset!(solver::LBFGSSolver)
+function SolverCore.reset!(solver::LBFGSSolver)
   reset!(solver.H)
 end
 
+function SolverCore.reset!(solver::LBFGSSolver, nlp::AbstractNLPModel{T, V}) where {T, V}
+  reset!(solver.H)
+  solver.h = LineModel(nlp, solver.x, solver.d)
```

Comment:
`solver.h` or `solver.H`?

Reply:
`h` is the model, and `H` the operator so it's fine.